python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -222,8 +254,8 @@ def resolve_partitionRunStats(self, graphene_info):
             numInProgress=num_in_progress,
             numSucceeded=num_succeeded,
             numFailed=num_failed,
-            numPartitionsWithRuns=len(by_partition_records),
-            numTotalRuns=len(records),
+            numPartitionsWithRuns=len(partition_run_data),
+            numTotalRuns=len(partition_run_data),  # hack, but this is unused
```

Comment:
more context on this? "is unused" gives me dangerous assumption vibes

Reply:
https://github.com/dagster-io/dagster/pull/8696 is the PR that rips out calls to `partitionRunStats`, `numTotalRuns`...

It's stacked on top of this one, so it's a bit of a chicken/egg issue.

The only other call site to this field is from `PartitionProgress`, which is unused and I'll rip out.  Once we land this, land the frontend changes, I'll remove the last frontend references to these graphql fields and then remove these resolvers.

Reply:
Update: https://github.com/dagster-io/dagster/pull/8796 rips out the remaining calls to numTotalRuns